src/main/java/tempest/ui/GUIManager.java

```diff
@@ -42,7 +33,7 @@
     private final ManageSessionsPage manageSessions;
     private final ManageModulesPage manageModules;
     private final DeleteSessionPage deleteSession;
-
+    private final DataProtectionPage dataProtection;
```

Comment:
We don't need a local variable of DataProtectionPage. The only reason some of the pages do have a local variable is so they can be used in the switch statement to update them.

So could you delete it and change the line below, to be the same as other pages that don't have a local variable

```java
dataProtection = new DataProtectionPage(state, this),
```